src/pages/index.js

```diff
@@ -1,30 +1,47 @@
-        In the meantime please subscribe to our newsletter to receive updates, or follow us on twitter: <a href="https://twitter.com/webaudioconf" target="_blank">@webaudioconference</a></p>
-      </div>
+    <div className="index__splash">
+      <h1 className="index__headline">
+        Web Audio Conf | Berlin
+      </h1>
+      <h4 className="index__headline">
+        19th - 21st September 2018
+      </h4>
+    </div>
+    <div className="index__welcome">
+      <p className="index__text">
+        The Web Audio Conference is an international conference dedicated to emerging audio-related web technologies such as Web Audio API, Web MIDI and Web RTC / WebGL.
+      </p>
+      <p className="index__text">
+        We welcome web developers, music technologists, computer musicians, application designers, researchers, and people involved in web standards.
+      </p>
+    </div>
+
+    <div className="index__moreInfo">
+      <h4>More info soon</h4>
+      <p className="index__text">
+        We are just getting started and will announce more information soon 📣.<br /> The date is set, so mark it in your calendar 📅✌️.
+        <br />In the meantime, please subscribe to our newsletter to receive updates, or follow us on twitter: <a href="https://twitter.com/webaudioconf" target="_blank">@webaudioconference</a>
+      </p>
+    </div>
+
+    <div className="index__newsletter">
       <div className="six columns">
-        <div dangerouslySetInnerHTML={{
-          __html: `
-          <form action="https://webaudioconf.us17.list-manage.com/subscribe/post?u=786ac969cb6e9a620eda77728&amp;id=84c4ce8f63" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" target="_blank" novalidate>
-            <input type="email" name="EMAIL" value="" id="mce-EMAIL" class="u-full-width" placeholder="E-mail address" />
-            <input type="submit" value="Subscribe" name="subscribe" class="button-primary" id="mc-embedded-subscribe" />
-            <div style="position: absolute; left: -5000px;" aria-hidden="true">
-              <input type="text" value="" name="b_786ac969cb6e9a620eda77728_84c4ce8f63" tabindex="-1" />
-            </div>
-        </form>`
-        }} />
+        <form action="https://webaudioconf.us17.list-manage.com/subscribe/post?u=786ac969cb6e9a620eda77728&amp;id=84c4ce8f63" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" target="_blank" noValidate>
+          <input type="email" name="EMAIL" id="mce-EMAIL" className="u-full-width" placeholder="E-mail address" />
+          <input type="submit" value="Subscribe" name="subscribe" className="button-primary" id="mc-embedded-subscribe" />
+          <div style={{ position: 'absolute', left: '-5000px' }} aria-hidden="true">
+            <input type="text" value="" name="b_786ac969cb6e9a620eda77728_84c4ce8f63" tabIndex="-1" />
+          </div>
+        </form>
       </div>
     </div>
-  </div>
+  </div >
```

Comment:
I couldn't find any other thing to complain about, which is why I have to comment on that extra whitespace here. :-) Otherwise it looks fine to me. Thanks @janmonschke